packages/components/src/box-control/icon.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+/**
+ * Internal dependencies
+ */
+import {
+	Root,
+	Viewbox,
+	TopStroke,
+	RightStroke,
+	BottomStroke,
+	LeftStroke,
+} from './styles/box-control-icon-styles';
+
+const BASE_ICON_SIZE = 24;
+
+export default function BoxControlIcon( {
+	size = 24,
+	sides = [ 'all' ],
+	...props
+} ) {
+	const top = getSide( sides, 'top' );
+	const right = getSide( sides, 'right' );
+	const bottom = getSide( sides, 'bottom' );
+	const left = getSide( sides, 'left' );
+
+	// Simulates SVG Icon scaling
+	const scale = size / BASE_ICON_SIZE;
+
+	return (
+		<Root style={ { transform: `scale(${ scale })` } } { ...props }>
+			<Viewbox>
+				<TopStroke isFocused={ top } />
+				<RightStroke isFocused={ right } />
+				<BottomStroke isFocused={ bottom } />
+				<LeftStroke isFocused={ left } />
+			</Viewbox>
+		</Root>
+	);
+}
+
+function getSide( sides, value ) {
+	const match = value.toLowerCase();
```

Comment:
Do we still need this? All apparent calls to `getSide` come with a lowercase `value`.

Reply:
Thank you for catching this! You are correct.

For context, `box-control/icon.js` was something I added earlier on. It was removed during mid-development, but added back again near the end, based on @jasmussen 's feedback.

Originally, it had the ability to highlight multiple sides, which is why it used an Array.

The case now is simpler :).

I've refactored and pushed it up!